deps/build.jl

```diff
@@ -122,27 +122,64 @@ end
 # need to be able to get the version before Python is initialized
 Py_GetVersion(libpy) = unsafe_string(ccall(Libdl.dlsym(libpy, :Py_GetVersion), Ptr{UInt8}, ()))
 
-const python = try
-    let py = get(ENV, "PYTHON", isfile("PYTHON") ? readchomp("PYTHON") : "python"), vers = convert(VersionNumber, pyconfigvar(py,"VERSION","0.0"))
-        if vers < v"2.7"
-            error("Python version $vers < 2.7 is not supported")
+# If the PYCALL_CURRENT_PROCESS is set, look for python symbols in the current
+# process first (e.g. for use from pyjulia).
+if get(ENV, "PYCALL_CURRENT_PROCESS", nothing) !== nothing
```

Comment:
Can't we just check whether the current process has the `Py_GetVersion` symbol, rather than using an environment variable?